src/ocean/io/serialize/StructSerializer.d

```diff
@@ -125,12 +127,8 @@ struct StructSerializer ( bool AllowUnions = false )
     ***************************************************************************/
 
     size_t dump ( S ) ( S* s, void delegate ( void[] data ) receive )
-    in
-    {
-        assertStructPtr!("dump")(s);
-    }
-    body
     {
+        verifyStructPtr!("dump")(s);
```

Comment:
Same for this module - it all should `enforce` with actual exception, those are not sanity violations